apps/url-encoder-decoder/src/App.tsx

```diff
@@ -0,0 +1,89 @@
+import React,{useState} from "react";
+import styled from "styled-components";
+import { isTriggered } from "@felvin-search/core";
+
+//------------Styled Components-------------
+// If you're unfamiliar with styled components
+// start here https://styled-components.com/docs/basics#getting-started
+
+const Button_enc = styled.button`
+  background: red;
+  width: 24%;
+  margin: 10px 5% 0 5%;
```

Comment:
It's also better to use units like `rem` when specifying width, margins or paddings.